requirements-dev.txt

```diff
@@ -10,4 +10,5 @@ pytest>=3.2.1
 pytest-pythonpath>=0.3
 pytest-watch==4.*
 tox>=1.8.0
+web3>=3.16.0
```

Comment:
Can you go ahead and pin this to the v4 beta.  Check the Releases section of the web3 repo for the exact version.